src/oc/storage/api/entries.clj

```diff
@@ -90,6 +90,11 @@
     
     true)) ; no existing entry, so this will fail existence check later
 
+(defn- valid-entry-revert? [entry-props]
+  (if (:revision-id entry-props)
```

Comment:
Would be better to check with schema validation here, like:
`(lib-schema/valid? schema/Int (:revision-id entry-props)`
WYT?

Reply:
👍